packages/flutter/lib/src/widgets/form.dart

```diff
@@ -478,6 +493,15 @@ class FormField<T> extends StatefulWidget {
   /// not an error is displayed, either wrap the  [TextFormField] in a fixed
   /// height parent like [SizedBox], or set the [InputDecoration.helperText]
   /// parameter to a space.
+  ///
+  /// Note: Using [forceErrorText] will bypasses the regular
```

Comment:
Another "Note": https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
I'll be removing this too.